addons/web/static/tests/views/fields/image_field_tests.js

```diff
@@ -861,4 +864,116 @@ QUnit.module("Fields", (hooks) => {
             );
         }
     );
+
+    QUnit.test(
+        "url should not use the record last updated date when the field is related",
+        async function (assert) {
+            serverData.models.partner.fields.related = {
+                name: "Binary",
+                type: "binary",
+                related: "user.image",
+            };
+
+            serverData.models.partner.fields.user = {
+                name: "User",
+                type: "many2one",
+                relation: "user",
+                default: 1,
+            };
+
+            serverData.models.user = {
+                fields: {
+                    image: {
+                        name: "Image",
+                        type: "binary",
+                    },
+                },
+                records: [
+                    {
+                        id: 1,
+                        image: "3 kb",
+                    },
+                ],
+            };
+
+            serverData.models.partner.records[0].__last_update = "2017-02-08 10:00:00";
+
+            const dates = [
+                deserializeDateTime("2017-02-06 10:00:00"), // 1486375200000
+                deserializeDateTime("2017-02-09 10:00:00"), // 1486634400000
+                deserializeDateTime("2017-02-09 11:00:00"), // 1486638000000
+            ];
+
+            patchWithCleanup(ImageField.prototype, {
+                _now: function () {
+                    assert.step("now");
+                    return dates.shift();
+                },
+            });
```

Comment:
Why not using `patchDate` instead?
I'm not a fan of adding a method like this just to patch it in the tests.